jerry-core/parser/js/js-parser-expr.c

```diff
@@ -2753,7 +3067,60 @@ parser_process_unary_expression (parser_context_t *context_p, /**< context */
       }
     }
   }
-} /* parser_process_unary_expression */
+} /* parser_process_prefix_unary_opcodes */
+
+/**
+ * Parse the postfix part of the expression
+ */
+static void
+parser_parse_postfix_expresion (parser_context_t *context_p, /**< context */
+                                size_t *grouping_level_p) /**< grouping level */
+{
+  /* Parse postfix part of a primary expression. */
+  while (true)
+  {
+    /* Since break would only break the switch, we use
+     * continue to continue this loop. Without continue,
+     * the code abandons the loop. */
+    switch (context_p->token.type)
+    {
+      case LEXER_DOT:
+      {
+        parser_parse_property_access (context_p);
+        continue;
+      }
+      case LEXER_LEFT_SQUARE:
+      {
+        parser_parse_element_access (context_p);
+        continue;
+      }
+      case LEXER_QUESTION_MARK_DOT:
+      {
+        parser_parse_optional_chain (context_p, grouping_level_p);
+        break;
```

Comment:
Based on the comment above I don't understand why is this a `break` statement rather than a `continue`?